crabs/tracker/utils/io.py

```diff
@@ -178,3 +181,149 @@ def release_video(video_output) -> None:
     """
     if video_output:
         video_output.release()
+
+
+def read_metrics_from_csv(filename):
+    """
+    Read the tracking output metrics from a CSV file.
+    To be called by plot_output_histogram.
+
+    Parameters
+    ----------
+    filename : str
+        Name of the CSV file to read.
+
+    Returns
+    -------
+    tuple:
+        Tuple containing lists of true positives, missed detections,
+        false positives, number of switches, and total ground truth for each frame.
+    """
+    true_positives_list = []
+    missed_detections_list = []
+    false_positives_list = []
+    num_switches_list = []
+    total_ground_truth_list = []
+    mota_value_list = []
+
+    with open(filename, mode="r") as file:
+        reader = csv.DictReader(file)
+        for row in reader:
+            true_positives_list.append(int(row["True Positives"]))
+            missed_detections_list.append(int(row["Missed Detections"]))
+            false_positives_list.append(int(row["False Positives"]))
+            num_switches_list.append(int(row["Number of Switches"]))
+            total_ground_truth_list.append(int(row["Total Ground Truth"]))
+            mota_value_list.append(float(row["Mota"]))
+
+    return (
+        true_positives_list,
```

Comment:
maybe this tuple can be a dict instead? It's a bit less of a code smell

Reply:
I think if we read the csv as a pandas dataframe instead we can extract the columns more efficiently (that is, without explicit looping).

There is also a dataframe [`.to_dict()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_dict.html) method, so we may be able to get the output dictionary in one go this way.